relayer/Cargo.toml

```diff
@@ -62,6 +62,7 @@ signature = "1.3.0"
 anyhow = "1.0.43"
 fraction = {version = "0.9.0", default-features = false }
 semver = "1.0"
+uint = "0.9"
```

Comment:
Should we watch out for dependency on `std`? It seems `std` is [enabled by default](https://github.com/paritytech/parity-common/tree/master/uint#crate-features).

Reply:
Good catch, but this is not an issue in the `ibc-relayer`, as we already depend a whole bunch on `std` and don't have plans currently to make the crate `no-std` compatible.

Reply:
Nice catch @adizere! And thanks for the clarification @romac! 🙏